### PR TITLE
Pluralise disaster import so the endpoint runs correctly

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Changes:

- Pluralise disaster import endpoint used by admin panel
- Add CUD handlers for dataSources

### CODE REVIEW

File: packages/meditrak-server/src/app/addRoutesToApp.js [hunk 1]
```diff
@@ -250,6 +254,7 @@ export function addRoutesToApp(app) {
   app.put('(/v[0-9]+)/surveyScreenComponents/:recordId', editSurveyScreenComponents);
   app.put('(/v[0-9]+)/answers/:recordId', editAnswers);
   app.put('(/v[0-9]+)/surveyResponses/:parentRecordId/answers/:recordId', editAnswers);
+  app.put('(/v[0-9]+)/dataSources/:recordId', editDataSources);
```
Comment (edmofro) [inline packages/meditrak-server/src/app/addRoutesToApp.js:257]: Does this work for editing data elements within the data groups dropdown (if that's possible)? The read uses something like dataSources/:dataGroupId/dataSources

File: packages/admin-panel/src/pages/resources/SocialFeedPage.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ const EDIT_CONFIG = {
 const CREATE_CONFIG = {
   title: 'Add Social Feed item',
   actionConfig: {
-    editEndpoint: 'feedItem',
+    editEndpoint: 'feedItems',
```
Comment (edmofro) [inline packages/admin-panel/src/pages/resources/SocialFeedPage.js:105]: This isn't mentioned in the description, but seems legit :+1:

**Review by edmofro — APPROVED** [review]

File: packages/meditrak-server/src/app/addRoutesToApp.js (hunk 1 above)
Comment (EMcQ-BES) [reply packages/meditrak-server/src/app/addRoutesToApp.js:257]: You can edit and delete via the dropdown, but those actions don't use a parent/child endpoint like the get does so this works for them

Comment (edmofro) [reply packages/meditrak-server/src/app/addRoutesToApp.js:257]: Grand

**Pull request merged**